box-browse-sdk/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -24,7 +24,7 @@
     <string name="box_browsesdk_see_additional_results">查看更多结果</string>
     <string name="box_browsesdk_search_hint">搜索</string>
     <string name="box_browsesdk_recent_searches_header">近期搜索</string>
-    <string name="box_browsesdk_search_results_header">&lt;B&gt;%1$s&lt;/b&gt; 中的结果</string>
```

Comment:
The original string is :
Results in <![CDATA[<b>%1$s</b>]]>

We seem to be losing the Cdata tags.